Sources/ArcGISToolkit/Components/UtilityNetworkTrace/UtilityNetworkTraceViewModel.swift

```diff
@@ -438,6 +448,10 @@ import SwiftUI
     
     /// Adds programatic starting points to the pending trace.
     private func addExternalStartingPoints() async {
+        pendingTrace.startingPoints.forEach { startingPoint in
+            deleteStartingPoint(startingPoint)
+        }
+        
```

Comment:
One open question here. When the external set of starting points changes should we clear all of the current starting points or only those also externally set?